src/components/transformation/create/Config/Header.tsx

```diff
@@ -0,0 +1,42 @@
+import { Typography, useTheme } from '@mui/material';
+import { useEditorStore_invalidEditors } from 'components/editor/Store/hooks';
+import { WarningCircle } from 'iconoir-react';
+import { FormattedMessage } from 'react-intl';
+import {
+    useTransformationCreate_emptySQLExists,
+    useTransformationCreate_shuffleKeyErrorsExist,
+} from 'stores/TransformationCreate/hooks';
+
+function DerivationCatalogHeader() {
+    const theme = useTheme();
+
+    // Draft Editor Store
+    const invalidEditors = useEditorStore_invalidEditors();
+
+    // Transformation Create Store
+    const emptySQLExists = useTransformationCreate_emptySQLExists();
+    const shuffleKeyErrorsExist =
+        useTransformationCreate_shuffleKeyErrorsExist();
+
+    return (
+        <>
+            {emptySQLExists ||
+            shuffleKeyErrorsExist ||
+            invalidEditors.length > 0 ? (
```

Comment:
Thinking... still need to decide on how to handle this situation. Since we only check the length we could just store off a simple "hasInvalidEditors" for this to reduce renders but not sure if the trade off is worth it. 

I think potentially that is the best approach once we get something wired into Zustand to handle computed values and then all those little "helper" store props are handled automagically for us.

Reply:
I was thinking the latter myself. Extending the store did not seem worthwhile because it is yet another related attribute to maintain that is (allegedly) relatively cheap.